### PR TITLE
Document Key management API for ext-mongodb 1.15

### PR DESCRIPTION
**Pull request by jmikola**
Moves the masterKey docs to a common snippet, since it's duplicated for createDataKey and rewrapManyDataKey.

Also adds missing docs for the KMIP provider's masterKey document, which was introduced in version 1.12.

https://jira.mongodb.org/browse/PHPC-2093
https://jira.mongodb.org/browse/PHPC-1912

### CODE REVIEW

File: reference/mongodb/mongodb/driver/clientencryption/rewrapmanydatakey.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns an object, which will have an optional
+   <literal>bulkWriteResult</literal> property containing the result of the
+   internal <literal>bulkWrite</literal> operation as an object. If no data keys
+   matched the filter or the write was unacknowledged, the
+   <literal>bulkWriteResult</literal> property will be &null;
```
Comment (cmb69) [inline reference/mongodb/mongodb/driver/clientencryption/rewrapmanydatakey.xml:93]:
```suggestion
   <literal>bulkWriteResult</literal> property will be &null;.
```

**Review by cmb69 — APPROVED** [review]
Thank you!

File: reference/mongodb/mongodb/driver/clientencryption/getkeybyaltname.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  <para>
+   Finds a single key document in the key vault collection with the given
+   alternate name <parameter>keyAltName</parameter>.
+  </para>
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <variablelist>
+<varlistentry>
```
Comment (alcaeus) [inline reference/mongodb/mongodb/driver/clientencryption/getkeybyaltname.xml:25]:
```suggestion
   <varlistentry>
```

**Review by alcaeus — APPROVED** [review]
Found a wrong indent, but docs LGTM.

**Comment by jmikola** [comment]
@cmb69: I didn't mean to open this on the main repo (just my fork for a private review) but thanks very much for the review!

**Pull request merged**